Makefile

```diff
@@ -1,7 +1,5 @@
 NAME=tendrl-commons
-VERSION := $(shell PYTHONPATH=. python -c \
-             'import tendrl.commons; print tendrl.commons.__version__' \
-             | sed 's/\.dev[0-9]*//')
+VERSION := 1.2.3
```

Comment:
This seems like an exception where we are hard coding the version

Why does the older makefile work for other repos? https://github.com/Tendrl/ceph-integration/blob/develop/Makefile

Can you remove __version__ from tendrl/commons/__init__.py and make it as it is for other repos via version.py ?